keycloak/themes/hmda/login/login.ftl

```diff
@@ -15,7 +15,7 @@
                     <#if usernameEditDisabled??>
                         <input id="username" name="username" type="text" autocapitalize="off" autocorrect="off" value="${(login.username!'')?html}" disabled>
                     <#else>
-                        <input id="username" name="username" value="${(login.username!'')?html}" type="text" autofocus autocomplete="off" />
+                        <input id="username" name="username" value="${(login.username!'')?html}" type="text" autofocus />
```

Comment:
I think I'm OK removing `autocomplete=off`.  It does provide some amount of added security, but I would argue probably not enough to warrant the usability ding.  It also sounds like [many browsers don't honor `autocomplete=off` for username fields](https://developer.mozilla.org/en-US/docs/Web/Security/Securing_your_site/Turning_off_form_autocompletion#The_autocomplete_attribute_and_login_fields) anyway.

All that said, don't be surprised if this doesn't come back up during security review.